src/pages/home-page/HomePage.js

```diff
@@ -1,26 +1,43 @@
+import { useSelector } from "react-redux";
+
 import logo from "assets/logo.png";
 import LoginForm from "components/users/LoginForm/LoginForm";
 
 import "./home-page.scss";
 
-const HomePage = () => (
-  <main className="home-page">
-    <section className="login">
-      <figure>
-        <img src={logo} alt="Logo" className="logo" />
-      </figure>
-      <LoginForm />
-      <div className="forgot-password-link regular-text">
-        <a href="/">I forgot my password.</a>
-      </div>
-    </section>
-    <section className="signup">
-      <p className="regular-text">Don't have an account?</p>
-      <button type="button" className="common-white-button regular-text">
-        Sign up
-      </button>
-    </section>
-  </main>
-);
+const HomePage = () => {
+  const loading = useSelector((state) => state.user.loading);
+
+  const loggedInUser = sessionStorage.getItem("user");
```

Comment:
Be careful with this line. You are mixing usage of redux with Session Storage. If you want to preserve de session you'll have to store data in your Session storage but user data could be read it from Redux where is easier to make changes from different parts of your app